apps/web/public/static/locales/en/common.json

```diff
@@ -562,6 +562,7 @@
   "leave": "Leave",
   "profile": "Profile",
   "my_team_url": "My team URL",
+  "my_teams": "My teams",
```

Comment:
To show when "Other teams" are shown, meaning you see teams you belong to and other ones, so it is clearer